util/b58-payloads/src/payloads.rs

```diff
@@ -1,11 +1,16 @@
 //! Payload encodings for use in QR codes and deep links.
 
 use crate::error::Error;
+
 use core::{convert::TryFrom, fmt};
-use crc::crc32;
+use std::str::FromStr;
+
 use mc_crypto_keys::{KeyError, RistrettoPublic};
 use mc_transaction_core::account_keys::{AccountKey, PublicAddress};
 use mc_transaction_std::identity::RootIdentity;
+use mc_util_uri::FogUri;
+
```

Comment:
Why did this move here? In most places we just group all use statements and let rustfmt order them.

Reply:
james has a pattern where he separates like `core/std` from `mc` from 3rd party libs, so i put crc at the bottom

the FogUri dep is new, I'm adding it to help validate the `fog_report_url` values below

Reply:
I group them in the cargo.toml, because "path" crates are special, but I just bunch all the "use" statements together as @eranrund is suggesting. The only ones I put elsewhere are things like `alloc::vec` where they get improperly merged...

Reply:
i see i got confused